onnxruntime/core/codegen/mti/tensor/gather_elements.cc

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/codegen/mti/tensor/gather_elements.h"
+
+#include "core/codegen/mti/mti_tvm_utils.h"
+#include <topi/transform.h>
+
+namespace onnxruntime {
+namespace tvm_codegen {
+
+tvm::Tensor GatherElements(const tvm::Tensor& t,
+                           int64_t axis,
```

Comment:
>axis [](start = 35, length = 4)

can this be negative? if not, maybe should use unsigned types?

Reply:
We always use int64_t for axis in other tensor apis. Seems it's better to follow the convention.